src/config.rs

```diff
@@ -3,6 +3,7 @@ use crate::tile_grid::SplitDirection;
 use std::io::{Error, ErrorKind};
 
 use std::str::FromStr;
+use yaml_rust::{ScanError, Yaml};

```

Comment:
Why do we need the ScanError here?

```suggestion
use yaml_rust::{Yaml};
```